src/bundle/Controller/ContentViewController.php

```diff
@@ -59,6 +59,29 @@ public function detailsTabAction(ContentView $view)
         return $view;
     }
 
+    public function versionsTabAction(ContentView $view)
+    {
+        $contentInfo = $view->getContent()->getVersionInfo()->getContentInfo();
+        $contentService = $this->getRepository()->getContentService();
+        $versions = $contentService->loadVersions($contentInfo);
+
+        $versionFilter = $this->container->get('ezsystems.platform_ui.hybrid.filter.version_filter');
```

Comment:
I'd prefer if we injected this somehow. I'll test some things with auto-wiring and I'll let you know.